### DISCUSSION TITLE
Unable to delete newline after a `}` block

### DISCUSSION THREAD

**Discussion by harry-xm** [redhat-developer/lsp4ij#689] [Q&A]
This does not seem like auto-formatting, because I could delete a newline from the next line.

Sample project: https://github.com/cloudflare/pingora/blob/e309436319ed5cbc3aaf53221070a1fd070b8bcf/pingora-limits/src/estimator.rs#L26

https://github.com/user-attachments/assets/d02b036e-31ff-484d-92e8-aa61e1d9dd0e

IDE info:
```
PyCharm 2024.3 (Community Edition)
Build #PC-243.21565.199, built on November 13, 2024
Runtime version: 21.0.5+8-b631.16 aarch64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.lwawt.macosx.LWCToolkit
macOS 15.1.1
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 3072M
Cores: 8
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
  i18n.locale=
Non-Bundled Plugins:
  com.redhat.devtools.lsp4ij (0.8.1)
  org.jetbrains.plugins.gitlab (243.21565.204)
```

**Comment by fbricon** [comment]
I can reproduce this issue on a .rs file, without having LSP4IJ installed, so looks like a bug in JetBrains products, I suggest you open a ticket upstream.

**Comment by angelozerr** [comment]
I close this discussion since it seems a bug from IntelliJ himself.
